net-libs/libomemo-c/libomemo-c-0.5.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="XMPP fork of Signal Protocol C Library supporting XEP-0384 OMEMO"
+HOMEPAGE="https://github.com/dino/libomemo-c/"
+SRC_URI="https://github.com/dino/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+KEYWORDS="amd64 arm64 x86"
```

Comment:
You should only include `~amd64` here, drop the keywords from the main package and request re-keywording through bugzilla once an update has been merged. 

Or we can use-mask omemo in x86 for the time being.

Also, no new packages straight to stable!

Reply:
Oops, really stupid mistake! Fixed.